project/src/main/webapp/create-volunteering-opportunity.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
Next time could you separate frontend change and backend change in different PRs?

Reply:
Will do, sorry about that

Reply:
Ack, will try to do this.